terra_layer/serializers.py

```diff
@@ -7,6 +10,12 @@
 from .models import CustomStyle, FilterField, Layer, Scene
 
 
+class BaseLayerSerializer(ModelSerializer):
```

Comment:
Should be in baselayer module ?

Reply:
Base layer module is not a terra module, and can be used in every project using mapbox in frontend. I dont think adding requirements to DRF in this module is a good idea. It is not required to write a serializer  for this, just using a reverse url for tilejson in another serializer (scene ?) should be sufficient

Reply:
https://github.com/Terralego/django-terra-geocrud/blob/master/terra_geocrud/serializers.py#L90

Reply:
https://github.com/Terralego/django-terra-geocrud/blob/master/terra_geocrud/views.py#L88

Reply:
It has been moved in another django app set as a dependency for this one (django-terra-settings).